core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/io/writeJson.kt

```diff
@@ -108,10 +116,45 @@ internal fun KlaxonJson.encodeValue(col: AnyCol, index: Int): Any? = when {
         } else v
     }
 
+    col.typeClass == BufferedImage::class -> col[index]?.let { image ->
```

Comment:
Hm, at first i didn't notice that given `ImageEncodingOptions(encodeAsBase64 = false)` BufferedImage will be serialized as `toString`. Maybe lift that condition up and, for example, if ImageEncodingOptions == null then call toString and otherwise proceed with encoding?

Reply:
refactored a bit